src/clusterfuzz/_internal/platforms/android/fetch_artifact.py

```diff
@@ -163,13 +168,34 @@ def get_client():
 
   return client
 
+def get_stable_build_info():
+  """Return stable artifact for cuttlefish branch and target."""
+  stable_build_info = STABLE_CUTTLEFISH_BUILD
+  gcs_build_info_url = DEFAULT_STABLE_CUTTLEFISH_BUILD_INFO
+
+  gcs_url_override = (
+    environment.get_value('OVERRIDE_STABLE_CUTTLEFISH_BUILD_INFO')
```

Comment:
Where is the `OVERRIDE_STABLE_CUTTLEFISH_BUILD_INFO` environment variable being defined?

I think overall we want to avoid adding any new environment variables.

It seems like this override would be a new, optional option that can be set, but I don't think we need to provide this option.